packages/p2p-v2/src/components/Table/Table.tsx

```diff
@@ -8,7 +8,7 @@ import './Table.scss';
 type TProps<T> = {
     columns?: ColumnDef<T>[];
     data: T[];
-    emptyDataMessage: string;
+    emptyDataMessage?: string;
```

Comment:
shouldn't this be required?

Reply:
no. the emptydatamessage case applies only for buyselltable. other tables don't have this issue. the myprofilecounterparties table can be replaced with deriv-com/ui table instead.

Reply:
yup not all tables show empty data message as they show an action screen without the headers